### PR TITLE
ragllm edb docs update

### PR DESCRIPTION
**Pull request by day0hero**
- **adding images for ragllm docs**
- **updated docs with edb info**
- **updating docs with edb info**

### CODE REVIEW

**Comment by mbaldessari** [comment]
This is an automated message:

You can preview this docs PR at http://445.docs-pr.validatedpatterns.io
Note that they get generated every five minutes, so please wait a bit.

File: content/patterns/rag-llm-gitops/getting-started.md [hunk 1]
```diff
@@ -30,6 +21,110 @@ cp values-secret.yaml.template ~/values-secret-rag-llm-gitops.yaml
...
+In Figure 5, we can see RAG augmented query. Llama 2 model is used for for language processing, LangChain to
+integrate different tools of the LLM-based application together and to process the PDF
+files and web pages, vector database such as PGVECTOR or REDIS, is used to store vectors, HuggingFace TGI is used to serve the Llama 2 model, Gradio is used for user interface and object storage to store language model and other datasets. Solution components are deployed as microservices in the Red Hat OpenShift cluster.
+
+#### Download diagrams
+View and download all of the diagrams above in our open source tooling site.
+
+[Open Diagrams](https://www.redhat.com/architect/portfolio/tool/index.html?#gitlab.com/osspa/portfolio-architecture-examples/-/raw/main/diagrams/rag-demo-vp.drawio)
+
+![Diagram](/images/rag-llm-gitops/diagram.png)
```
Comment (swati-kale) [inline content/patterns/rag-llm-gitops/getting-started.md:54]: This demo arch/Fig 6 diagram needs to be changed as it has reference to REDIS to Fig 6

Comment (day0hero) [reply content/patterns/rag-llm-gitops/getting-started.md]: 👍

File: content/patterns/rag-llm-gitops/getting-started.md [hunk 2]
```diff
@@ -39,15 +134,15 @@ cp values-secret.yaml.template ~/values-secret-rag-llm-gitops.yaml
...
-Note: If the hf-text-generation-server is not running, make sure you have followed the steps to configure a node with GPU from the [instructions](/rag-llm-gitops/gpuprovisioning) provided above.
+Note: If the hf-text-generation-server is not running, make sure you have followed the steps to configure a node with GPU from the [instructions](../gpu_provisioning) provided above.
 
 ### 2: Launch the application
 
-- Navigate to routes, Networking --> Routes
+- Click the `Application box` icon in the header, and select `Retrieval-Augmented-Generation (RAG) LLM Demonstration UI`
 
-  ![Routes](/images/rag-llm-gitops/routes.png)
+![Launch Application](/images/rag-llm-gitops/launch-application.png)
```
Comment (sauagarwa) [inline content/patterns/rag-llm-gitops/getting-started.md:143]: The images are not showing up

File: content/patterns/rag-llm-gitops/getting-started.md [hunk 3]
```diff
@@ -30,6 +21,110 @@ cp values-secret.yaml.template ~/values-secret-rag-llm-gitops.yaml
...
+- **Hugging Face Text Generation Inference Server:** The pattern deploys a Hugging Face TGIS server. The server deploys `mistral-community/Mistral-7B-v0.2` model. The server will require a GPU node.
+- **EDB (PGVECTOR) / Redis Server:** A Vector Database server is deployed to store vector embeddings created from Red Hat product documentation.
+- **Populate VectorDb Job:** The job creates the embeddings and populates the vector database.
+- **LLM Application:** This is a Chatbot application that can generate a project proposal by augmenting the LLM with the Red Hat product documentation stored in vector db.
+- **Prometheus:** Deploys a prometheus instance to store the various metrics from the LLM application and TGIS server.
+- **Grafana:** Deploys Grafana application to visualize the metrics.
+
+## Deploying the demo
+
+### Cloning repository
```
Comment (sauagarwa) [inline content/patterns/rag-llm-gitops/getting-started.md:69]: We should mention which repository to fork first

File: content/patterns/rag-llm-gitops/getting-started.md [hunk 4]
```diff
@@ -30,6 +21,110 @@ cp values-secret.yaml.template ~/values-secret-rag-llm-gitops.yaml
...
+
+- **Hugging Face Text Generation Inference Server:** The pattern deploys a Hugging Face TGIS server. The server deploys `mistral-community/Mistral-7B-v0.2` model. The server will require a GPU node.
+- **EDB (PGVECTOR) / Redis Server:** A Vector Database server is deployed to store vector embeddings created from Red Hat product documentation.
+- **Populate VectorDb Job:** The job creates the embeddings and populates the vector database.
+- **LLM Application:** This is a Chatbot application that can generate a project proposal by augmenting the LLM with the Red Hat product documentation stored in vector db.
+- **Prometheus:** Deploys a prometheus instance to store the various metrics from the LLM application and TGIS server.
+- **Grafana:** Deploys Grafana application to visualize the metrics.
+
+## Deploying the demo
+
```
Comment (sauagarwa) [inline content/patterns/rag-llm-gitops/getting-started.md:68]: ### Forking repository
Fork repository https://github.com/validatedpatterns/rag-llm-gitops.git

File: content/patterns/rag-llm-gitops/_index.md [hunk 5]
```diff
@@ -19,31 +19,36 @@ links:
...
+## Pre-requisites
+
+- Podman
+- Red Hat Openshift cluster running in AWS. Supported regions are us-west-2 and us-east-1.
+- GPU Node to run Hugging Face Text Generation Inference server on Red Hat OpenShift cluster.
+- Create a fork of the [rag-llm-gitops](https://github.com/validatedpatterns/rag-llm-gitops.git) git repository.
+
+## Demo Description & Architecture
 
 The goal of this demo is to demonstrate a Chatbot LLM application augmented with data from Red Hat product documentation running on Red Hat OpenShift. It deploys an LLM application that connects to multiple LLM providers such as OpenAI, Hugging Face, and NVIDIA NIM. The application generates a project proposal for a Red Hat product
```
Comment (sauagarwa) [inline content/patterns/rag-llm-gitops/_index.md:37]: The application generates a project proposal for a Red Hat product. (Missing .)

File: content/patterns/rag-llm-gitops/getting-started.md [hunk 6]
```diff
@@ -30,6 +21,110 @@ cp values-secret.yaml.template ~/values-secret-rag-llm-gitops.yaml
...
+
+#### RAG Augmented Query
+
+
+![query](https://gitlab.com/osspa/portfolio-architecture-examples/-/raw/main/images/schematic-diagrams/rag-demo-vp-query-sd.png)
+
+_Figure 5. Schematic diagram for RAG demo augmented query._
+
+In Figure 5, we can see RAG augmented query. Llama 2 model is used for for language processing, LangChain to
```
Comment (sauagarwa) [inline content/patterns/rag-llm-gitops/getting-started.md:45]: two for for

**Pull request merged**